.github/workflows/tests.yml

```diff
@@ -7,122 +7,13 @@ on:
-          name: bazel-testlogs
-          path: bazel-testlogs
-
-  build_arm64_simulator:
-    name: Build arm64 Simulator
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Select Xcode
-        run: .github/workflows/xcode_select.sh
-      - name: Build and Test
-        run: |
-          bazelisk build --ios_multi_cpus=sim_arm64  --features apple.virtualize_frameworks -- //... -//tests/ios/...
-
-          # Misc issues:
-          # Carthage is busted for -//tests/ios/frameworks/sources-with-prebuilt-binaries/...
-          # Fails on a non fat framework for //tests/ios/unit-test/test-imports-app/
-          bazelisk build --ios_multi_cpus=sim_arm64 --features apple.virtualize_frameworks \
-              --apple_platform_type=ios \
-              --deleted_packages='' \
-              -- //tests/ios/... \
-              -//tests/ios/frameworks/sources-with-prebuilt-binaries/... \
-              -//tests/ios/unit-test/test-imports-app/...
-
-      - uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: bazel-testlogs
-          path: bazel-testlogs
 
-  buildifier:
-    name: Check Starlark and Docs
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Select Xcode
-        run: .github/workflows/xcode_select.sh
-        # Note: we need to pass the absolute to the Bazel run
-      - name: buildifier
-        run: find $PWD -type f \( -name 'WORKSPACE' -o -name '*.bzl' -o -name '*.bazel' \) | xargs bazel run buildifier -- -lint=fix && git diff --exit-code
-      - name: Check docs
-        run: bazelisk run docs && git diff --exit-code docs
-  xcodeproj_tests_xcode_12_5_1:
-    name: .xcodeproj Tests on Xcode 12.5.1
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Select Xcode 12.5.1
-        run: .github/workflows/xcode_select.sh
-      - name: Run tests
-        run: ./tests/xcodeproj-tests.sh --clean
-
-  lldb_ios_tests_xcode_12_5_1:
-    name: LLDB tests on Xcode 12.5.1
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v2
-      - name: Select Xcode 12.5.1
-        run: .github/workflows/xcode_select.sh
-      - name: Run tests
-        run: bazelisk test tests/ios/lldb/app:objc_app_po_test  tests/ios/lldb/app:objc_app_variable_test --config lldb_ios_test
-      - uses: actions/upload-artifact@v2
-        if: failure()
-        with:
-          name: bazel-testlogs
-          path: bazel-testlogs
-  multi_arch_support:
+  reproduce_github_issue:
     name: Build iOS App for Multiple Architecture
     runs-on: macos-latest
     steps:
       - uses: actions/checkout@v2
       - name: Select Xcode
         run: .github/workflows/xcode_select.sh
       - name: Build App
-        run: bazelisk build -s tests/ios/app/App --apple_platform_type=ios --ios_minimum_os=10.2  --ios_multi_cpus=i386,x86_64
+        run: bazelisk build -s tests/ios/app/App --ios_minimum_os=13
```

Comment:
I think it fails with the `.0` suffix. 
`bazelisk build -s tests/ios/app/App --ios_minimum_os=13.0`